README.md

```diff
@@ -116,19 +116,19 @@ Service calls take the following custom parameters, all as strings, (required wh
         * role without "roles/" prefix (see https://cloud.google.com/iam/docs/understanding-roles for available roles)
 * [Cloud Storage](https://cloud.google.com/storage/docs/)
     * Provison
-        * name (required)
+        * name (optional)
         * location (for options, see https://cloud.google.com/storage/docs/bucket-locations. Defaults to us)
     * Bind
         * role without "roles/" prefix (see https://cloud.google.com/iam/docs/understanding-roles for available roles)
 * [BigQuery](https://cloud.google.com/bigquery/docs/)
     * Provison
-        * name (required)
+        * name (optional)
     * Bind
         * role without "roles/" prefix (see https://cloud.google.com/iam/docs/understanding-roles for available roles), e.g. pubsub.admin
 * [CloudSQL](https://cloud.google.com/sql/docs/)
     * Provison
-        * instance_name (required)
-        * database_name (required)
+        * instance_name (defaults to a generated value)
```

Comment:
nitpick: these hints say different things than the other hints. Would standardize on one. Not going to let this hold up the merge though.